### PR TITLE
create a loganalytics sample

### PR DESCRIPTION
**Pull request by kazrael2119**
https://github.com/Azure/azure-sdk-for-python/issues/23862

### CODE REVIEW

File: samples/loganalytics/README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+1.  If you don't already have it, [install Python](https://www.python.org/downloads/).
+
+    This sample (and the SDK) is compatible with Python 3.6+.
+
+2.  General recommendation for Python development is to use a Virtual Environment.
+    For more information, see https://docs.python.org/3/tutorial/venv.html
+
+    Install and initialize the virtual environment with the "venv" module on Python 3 (you must install [virtualenv](https://pypi.python.org/pypi/virtualenv) for Python 2.7):
+
```
Comment (msyyc) [inline samples/loganalytics/README.md:47]: python sdk does not support py2.7 anymore

**Review by msyyc — CHANGES_REQUESTED** [review]
try to use `black` to format code style

File: samples/loganalytics/manage_workspace.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    loganalytics=loganalytics_client_gov.workspaces.begin_create_or_update(
+        GROUP_NAME,
+        workspace_name,
+        {"location": "eastus"}
+    )
+    print("Create consumption:\n{}\n".format(loganalytics))
+
+    loganalytics=loganalytics_client_gov.workspaces.list()
+    print("List consumption:\n{}\n".format(loganalytics))
```
Comment (BigCat20196) [inline samples/loganalytics/manage_workspace.py:32]: This result returns an iterator.

File: samples/loganalytics/manage_workspace_with_gov_cloud.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    resource_client.resource_groups.create_or_update(
+        GROUP_NAME,
+        {"location": "eastus"}
+    )
+
+    loganalytics = loganalytics_client_gov.workspaces.begin_create_or_update(
+        GROUP_NAME,
+        workspace_name,
+        {"location": "eastus"}
+    )
```
Comment (BigCat20196) [inline samples/loganalytics/manage_workspace_with_gov_cloud.py:31]:
```suggestion
    ).result()
```

File: samples/loganalytics/manage_workspace_with_gov_cloud.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    loganalytics = loganalytics_client_gov.workspaces.get(
+        GROUP_NAME,
+        workspace_name
+    )
+    print("\nGet consumption:\n{}\n".format(loganalytics))
+
+    loganalytics = loganalytics_client_gov.workspaces.begin_delete(
+        GROUP_NAME,
+        workspace_name
+    )
```
Comment (msyyc) [inline samples/loganalytics/manage_workspace_with_gov_cloud.py:48]:
```suggestion
    ).result()
```

File: samples/loganalytics/manage_workspace.py [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+        GROUP_NAME,
+        workspace_name
+    )
+    print("\nGet consumption:\n{}\n".format(loganalytics))
+
+
+    loganalytics=loganalytics_client_gov.workspaces.begin_delete(
+        GROUP_NAME,
+        workspace_name
+    )
```
Comment (msyyc) [inline samples/loganalytics/manage_workspace.py:46]:
```suggestion
    ).result()
```

**Review by msyyc — APPROVED** [review]

**Pull request merged**